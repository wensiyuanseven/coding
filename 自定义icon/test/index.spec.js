import { mount } from '@vue/test-utils'
import DemoButton from '../../demo-button'
// https://vue-test-utils.vuejs.org/zh/
// https://juejin.cn/post/6930475738646708237
// https:// juejin.cn/post/6844904196244766728
test('test icon', () => {
    const wrapper = mount(DemoButton)
    expect(wrapper).toMatchSnapshot()
})
