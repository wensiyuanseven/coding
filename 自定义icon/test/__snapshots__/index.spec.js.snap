// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render demo button 1`] = `<button class="demo-button"></button>`;

exports[`测试按钮 1`] = `<button class="demo-button"></button>`;
